types/v56/nsxv_types.go

```diff
@@ -406,3 +406,45 @@ type EdgeDhcpRelayAgent struct {
 type EdgeDhcpRelayAgents struct {
 	Agents []EdgeDhcpRelayAgent `xml:"relayAgent"`
 }
+
+// EdgeDhcpLease holds a list of EdgeDhcpLeaseInfo
+type EdgeDhcpLease struct {
+	XMLName        xml.Name             `xml:"dhcpLeaseInfo"`
+	DhcpLeaseInfos []*EdgeDhcpLeaseInfo `xml:"leaseInfo"`
+}
+
+// EdgeDhcpLeaseInfo contains information about DHCP leases provided by NSX-V edge gateway
+type EdgeDhcpLeaseInfo struct {
```

Comment:
Purely out of curiosity, are this structure's elements also order-dependent like we observed in some other places?

Reply:
This is a read-only endpoint I think there is no update option.